### PR TITLE
Update verify-egress command

### PR DESCRIPTION
**Pull request by fahlmant**
With the merge of https://github.com/openshift/osd-network-verifier/pull/122, the interface for running an egress test has changed. This PR updates the verify-egress command to use the latest interface, as well as expose new options available for the test.

### CODE REVIEW

**Review by fahlmant — COMMENTED** [review]
/hold for review

File: cmd/network/verification.go [hunk 1]
```diff
@@ -63,6 +71,12 @@ osdctl network verify-egress --subnet-id $(SUBNET_ID) --region $(AWS_REGION)`,
 	validateEgressCmd.Flags().BoolVar(&config.debug, "debug", false, "(optional) if true, enable additional debug-level logging")
 	validateEgressCmd.Flags().DurationVar(&config.timeout, "timeout", 1*time.Second, "(optional) timeout for individual egress verification requests")
 	validateEgressCmd.Flags().StringVar(&config.kmsKeyID, "kms-key-id", "", "(optional) ID of KMS key used to encrypt root volumes of compute instances. Defaults to cloud account default key")
+	validateEgressCmd.Flags().StringVarP(&config.awsProfile, "profile", "p", "", "AWS Profile")
```
Comment (mjlshen) [inline cmd/network/verification.go:74]: I think we should add `(optional)` into the flag descriptions of profile and security-group

**Comment by rafael-azevedo** [comment]
/lgtm 

tested locally all working well

**Comment by clcollins** [comment]
/lgtm
/approve 

Great work!

**Pull request merged**